packages/core/src/store/new-store/store.ts

```diff
@@ -57,58 +61,83 @@ export const useDndKitStore = create<DndKitStore>((set, get) => {
       });
     },
     registerDroppable: (payload: RegisterDroppableActionPayload) => {
-      const state = get();
       const {element} = payload;
       const {id} = element;
-      const containers = new DroppableContainersMap(state.droppable.containers);
-      containers.set(id, element);
+      droppablesToRegister.push(id, element);
+      if (droppablesToRegister.length !== 2) {
+        return;
+      }
```

Comment:
musste grad erst einmal den Rest des PRs lesen, um diese Stelle zu verstehen
ist das paarweise hinzufügen in ein flaches Array notwendig oder ließe sich das auch mit einem Tuple im Array abbilden?

Reply:
Das ließe sich auch mit einem Tupel abbilden, allerdings weiß ich nicht, wie viel performance es kostet, 10k Arrays zu erstellen.

Das rauszufinden könnte eine weile dauern, deshalb hab ich's erstmal so gemacht

Reply:
ok kann ich nachvollziehen - magst du wahlweise die Stellen, die außerhalb der Zählschleife liegen oder zumindest das Array selbst dann einmal um einen Kommentar ergänzen?

Reply:
sehr gerne, einen Moment

Reply:
@5ebastianMeier done